docs/tutorials/field_type/3_implement_the_tweet_type_class.md

```diff
@@ -256,23 +252,30 @@ public function fromPersistenceValue(PersistenceValue $fieldValue)
 
 ## Fetching data from the Twitter API
 
-As explained in the tutorial's introduction, you will enrich our tweet's URL with the embed version, fetched using the Twitter API. To do so, you will, when `toPersistenceValue()` is called, fill in the value's contents property from this method, before creating the `PersistenceValue` object.
+As explained in the tutorial's introduction, you will enrich the tweet's URL with the embed version, fetched using the Twitter API. To do so, you will, when `toPersistenceValue()` is called, fill in the value's contents property from this method, before creating the `PersistenceValue` object.
```

Comment:
this method -> not sure which method, toPersistenceValue()?

To do so, you will fill in the value's contents property from `toPersistenceValue(`) method when it is called, before creating the `PersistenceValue` object?